### DISCUSSION TITLE
Adding new columns doesnt add Meta data category to the note

### DISCUSSION THREAD

**Discussion by jasmeetchadha** [RafaelGB/obsidian-db-folder#131] [Q&A]
Hi - once I have a DB folder note in the folder, if I add a new column and then add a value in that column for my file
1) Sometimes I am unable to do that (i.e. add the value) 
2) Sometimes when I am able to add the value in to the DB folder page... it doesnt show in the note. I was hoping it would append that new metadata value in the note.

Can you please help?

Thanks

**Comment by jasmeetchadha** [comment]
Actually, I realize I am unable to set changes for many notes from the Obsidian DB file. Do not understand why? 

Any help / idea

**Comment by RafaelGB** [comment]
Hi @jasmeetchadha  , maybe your note have somethibg that breaks your yaml, cuald you share the note that don’t get your editions?

**Reply by jasmeetchadha** [reply]
The yaml works fine inside the note. I will send you the note if it helps. 

Here or some where else?

**Comment by RafaelGB** [comment]
here is okey :) or just a screenshot

**Reply by jasmeetchadha** [reply]
So I am trying to change the "THEME" meta item from the DB table and I cannot add it. If I try to change any other meta-item (such as Bucket) to another value, I cannot either.


```

---
# Context related to the KNOWLEDGE
Process: 
Source: 
GQV: 

# Context related to the Note
Theme: 
Sector: tech
Team: 
Bucket: Type1
Holding: Core
Folio: list
Scenarios: 1/2/++

# Context related to the EVERYNOTE
Note: #note 
Status: 
Eisen: 

Date:: 2022-05-09
---
Summary:: summary of note

Related idea:
Context: 


People: 
```

**Comment by RafaelGB** [comment]
Date:: 2022-05-09

have two ‘:’  and that breaks the yaml

I didnt test the yaml with comments, could be a problem too but it shouldn’t

**Reply by jasmeetchadha** [reply]
Changed. Still doesnt work :(

**Reply by jasmeetchadha** [reply]
I also cant change any meta data for any of the other files. I was able to do that before. Add/remove/change

May I try uninstalling and re-installing DB folder?

**Reply by jasmeetchadha** [reply]
So I did some troubleshooting. 

1) If I have a note with YAML pre-populated, the DB folder note will pull in the values, but I cannot edit it... especially if there are YAML meta fields with no values

2) Then if I remove all the meta fields with blank fields, it becomes editable

3) Then if I delete the meta fields from now editable note in DB folder. Those changes get updated in the note. Removing / changing / adding values or even adding new columns/meta fields.

The issue is 1 and the need to do 2, to fix it. 

Is it something that can be fixed on your end or I will have to rewrite the YAML for all notes again? 

Thanks

**Reply by RafaelGB** [reply]
I'm sorry for the inconvenience you are having with the use of the plugin.  It seems that those empty characters prevented receiving the info correctly and then being able to know what to edit.  In the future we will improve that reading and I note your incidence.  I can't think of any other solution at the moment.

**Reply by jasmeetchadha** [reply]
Oh no, please don’t apologise. It’s a work in progress. I am just sharing it to help iterate and solve it. Not complaining at all :) 

it’s a beautiful:)
